src/Grid/Service/GridService.php

```diff
@@ -268,4 +272,13 @@ private function getColumnResolvers(): array
 
         return $this->columnResolvers;
     }
+
+    private function isExportable(string $type): bool
+    {
+        if (!array_key_exists($type, $this->getColumnDefinitions())) {
+            return false;
```

Comment:
I would not throw it here i would do it instead of the `continue`. 

It would be a better feedback for the user that he knows why some columns are not in the exported file... So i would work with an exception here.

Reply:
Sounds good to me!